Services/BrokerService/Utils/BrokerFactory.cs

```diff
@@ -1,13 +1,28 @@
-﻿using System;
+﻿using BrokerService;
+using DTO;
 using Interfaces;
+using Kernel.CustomExceptions;
+using TinkoffIntegrationLib;
+using Microsoft.Extensions.Logging;
+using BrokerService.Controllers;
 
 namespace BrokerServices.Utils
 {
-    public static class BrokerFactory
+    internal static class BrokerFactory
```

Comment:
Why we use internal here? Will there be problems with testing for these reason?

Reply:
> Why we use internal here? Will there be problems with testing for these reason?

It makes no sense to test such simple logic.

Reply:
> Why we use internal here? Will there be problems with testing for these reason?

I changed the access modifier.